modules/cli/src/main/scala/scala/cli/commands/package0/Package.scala

```diff
@@ -194,43 +194,7 @@ object Package extends ScalaCommand[PackageOptions] with BuildCommandHelpers {
           .map(_ => None)
       }
     else {
-      val packageType: PackageType = value {
-        val basePackageTypeOpt = build.options.notForBloopOptions.packageOptions.packageTypeOpt
-        lazy val validPackageScalaJS =
-          Seq(PackageType.LibraryJar, PackageType.SourceJar, PackageType.DocJar)
-        lazy val validPackageScalaNative =
-          Seq(PackageType.LibraryJar, PackageType.SourceJar, PackageType.DocJar)
-
-        forcedPackageTypeOpt -> build.options.platform.value match {
-          case (Some(forcedPackageType), _) => Right(forcedPackageType)
-          case (_, _) if build.options.notForBloopOptions.packageOptions.isDockerEnabled =>
-            for (basePackageType <- basePackageTypeOpt)
```

Comment:
Let's improve this code while we're touching it.
This for loop has no effect, the result of this `case` will always be `Right(PackageType.Docker)`